### PR TITLE
Android Image Cropper added

### PR DESCRIPTION
**Pull request by Rainer-Lang**

### CODE REVIEW

File: library/src/main/res/values/library_android_image_cropper_strings.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<resources>
+	<string name="define_avatar_view"></string>
```
Comment (mikepenz) [inline library/src/main/res/values/library_android_image_cropper_strings.xml]: "_int" fehlt :D

**Comment by Rainer-Lang** [comment]
changed

File: library/src/main/res/values/library_android_image_cropper_strings.xml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<resources>
+	<string name="define_int_avatar_view"></string>
```
Comment (mikepenz) [inline library/src/main/res/values/library_android_image_cropper_strings.xml]: oh one more thing. the identifier "avatar_view" does not make much sense for this lib. In addition it duplicates with the existing "avatar_view" library

**Comment by mikepenz** [comment]
Now it's happy. AWESOME JOB 👍

**Comment by Rainer-Lang** [comment]
thanks

**Pull request merged**